# Height distance from robot's feet to camera (unit=m)
camera_robot_height: 1.5

# Birds eye view width of the bounding box to modify (unit=m)
box_width: 1.0

# Birds eye view height of the bounding box to modify (unit=m)
box_front: 1.5

# Birds eye view height of the bounding box to modify (unit=m)
box_back: 0.2

# Depth (down) of bounding box from origin z=0 (unit=m)
box_bottom: -0.5

# Height (up) of bounding box from origin z=0 (unit=m)
box_top: 0.3

# Starting/minimum value of pixels of the depth image sent to the service
image_height_base: 10